.github/workflows/validate.yaml

```
@@ -0,0 +1,72 @@
+name: validate
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - 'v**'
+  pull_request:
+    branches:
+      - master
+      - 'v**'
+
+jobs:
+  #
+  # golangci-lint
+  #
+  linters:
+    name: Linters
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 10
+
+    strategy:
+      matrix:
+        go-version: [1.16.3]
+        # TODO Fix linting on Windows (fails because of autogen)
+        # typechecking error: D:\a\docker\docker\src\github.com\docker\docker\cmd\dockerd\docker_windows.go:8:4:
+        # could not import github.com/docker/docker/autogen/winresources/dockerd
+        # os: [ubuntu-20.04, windows-2019]
+        os: [ubuntu-20.04]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: src/github.com/docker/docker
+      - name: install dependencies
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt -q update
+          sudo apt -q install \
```

These are probably printing `WARNING: apt does not have a stable CLI interface. Use with caution in scripts.`, so I'd suggest using `apt-get` here instead:

```suggestion
          sudo apt-get -q update
          sudo apt-get -q install \
```

---

Damn! Looks like I was caught red-handed doing a copy/pasta from (don't recall where) 😂 I'll update